src/Core/Domain/Entity/Setting.php

```diff
@@ -154,4 +155,36 @@ public function getInitializer(): ?string
     {
         return $this->initializer;
     }
+
+    /**
+     * @return bool
+     */
+    public function isSdk(): bool
+    {
+        return $this->getSettingType() === SettingEnum::SETTING_TYPE_SDK;
+    }
+
+    /**
+     * @return bool
+     */
+    public function isProject(): bool
```

Comment:
need to align it with the interface in the contracts. Looks like it is removed there. Also, it can not be simply changed in a one move. Please check my comment https://github.com/spryker-sdk/sdk-contracts/pull/19#discussion_r952557617

Reply:
done